app/presenters/work_provenance_display.rb

```diff
@@ -0,0 +1,33 @@
+
+# Display the provenance of a work on the front end:
+# WorkProvenanceDisplay.new(work).display
+
+class WorkProvenanceDisplay < ViewModel
+  valid_model_type_names 'Work'
+  def display
+    return "" if model.provenance.blank?
+    @provenance_summary, @provenance_notes = split_provenance
+    render "/presenters/work_provenance", model: model, view: self
+  end
+
+  def provenance_summary
+    DescriptionDisplayFormatter.new(@provenance_summary).format
+  end
+
+  def provenance_notes
+    DescriptionDisplayFormatter.new(@provenance_notes).format
+  end
+
+
+  # To split the provenance field into regular provenance info and a set of notes,
+  # use as a delimiter the string NOTES, in all caps, followed by an optional colon.
+  #
+  # The regular expression also captures extra blank space
+  # before and after the match, as well as before and after the word "NOTES".
+
```

Comment:
eliminate empty line between comments and method. In addition to being consistent style, the empty line would prevent the `yard` automatic documentation generator from realizing the comments are documenting the method.